cinema/serializers.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=255, required=True)
```

Comment:
The `description` field in the serializer has a `max_length` of 255, but the corresponding model field is a `TextField`, which does not have a `max_length` constraint. Consider removing the `max_length` from the serializer to align with the model definition.